test/topology_experimental_raft/test_topology_upgrade_not_stuck_after_recent_removal.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import logging
+import pytest
+import time
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.pylib.util import wait_for_cql_and_get_hosts
+from test.topology.conftest import skip_mode
+from test.topology.util import wait_until_topology_upgrade_finishes, \
+        wait_for_cdc_generations_publishing, \
+        check_system_topology_and_cdc_generations_v3_consistency
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_topology_upgrade_not_stuck_after_recent_removal(request, manager: ManagerClient):
```

Comment:
Please use the jenkins byo job to run the new test in both modes on both architectures 100 times

Reply:
Hopefully, the CI will take care of that after I push an updated version of the PR.